app/models/lesson.rb

```diff
@@ -25,14 +25,14 @@ def needs_update
   def publish
     twitters = Twitter_API.new
     client = twitters.client
-    
+
     tweet_id = 0;
     tweets.order(:id).each do |tweet|
       tweet_id = tweet.publish_tweet(client, tweet_id)
     end
 
     update(approved: true,
-    	thread_link: "https://twitter.com/tweechable/status/#{tweets.first.twitter_id}", 
+    	thread_link: "https://twitter.com/tweechable/status/#{tweets.first.tweet_index}",
```

Comment:
I'm pretty sure this should keep using `twitter_id` instead of `tweet_index` - twitter_id is the id the tweet on twitter and thread_link needs to point to that so it can link to them when making posts. This would set all the thread_links to /0, wouldn't it?